ui/app/styles/utils/_bulma_variables.scss

```diff
@@ -70,7 +431,6 @@ $breadcrumb-item-active-color: $black;
 $navbar-background-color: transparent;
 
 $menu-item-hover-background-color: $blue;
-$menu-item-hover-color: $white;
```

Comment:
I hate this file. Yes. We override this to the color of $text, on row 441. So we removed a duplicate. Granted it would be better to remove all duplicates at once, so i'll go ahead and put this back in and make a ticket to remove.

```
//menu
$menu-item-hover-color: $text;
```